.github/workflows/push-tests.yml

```diff
@@ -9,14 +9,21 @@ on:
 
 jobs:
   lint:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
-      - name: Get yarn cache directory path
```

Comment:
is there a reason we're not caching the directory anymore?

Reply:
```
- name: Set up Node.js
  uses: actions/setup-node@v3
  with:
    node-version: '16'
    cache: 'yarn'

- name: Install project dependencies
  run: yarn
  
```
  
`actions/setup-node@v2` or newer has caching built-in, so we no longer need to set up actions/cache. Isn't that cool? 😬😯